pkg/cmd/operator/cmd.go

```diff
@@ -71,7 +71,7 @@ func NewOperator() *cobra.Command {
 			log.Info("setting up manager")
 			mgr, err := manager.New(cfg, manager.Options{
 				MetricsBindAddress:      ":2112",
-				LeaderElection:          true,
+				LeaderElection:          false,
```

Comment:
Was this an intentional change? Or something left in from testing?

Reply:
Nope, just testing stuff that snuck in. Good catch!